app/routers/weight.py

```diff
@@ -0,0 +1,65 @@
+from typing import Union
+
+from fastapi import APIRouter, Depends, Request
+from sqlalchemy.orm import Session
+from starlette.responses import RedirectResponse
+
+from app.database.models import User
+from app.dependencies import get_db
+from app.dependencies import templates
+
+
+router = APIRouter(tags=["weight"],)
+
+
+@router.get("/weight")
+async def weight_form(
```

Comment:
function names start with a verb and they describe what the function does

Reply:
Done